test/webdriver.js

```diff
@@ -63,6 +64,16 @@ describe('startDriver', () => {
 		assert.equal(service.executable_, chromedriver.path);
 	});
 
+	it('uses the passed in chromedriver path with chrome-headless', async () => {
+		browser = 'chrome-headless';
+		config.chromedriverPath = path.relative(process.cwd(), chromedriver.path);
+		await startDriver(config);
+		const service = chrome.getDefaultService();
+
+		assert.notEqual(config.chromedriverPath, chromedriver.path);
+		assert.equal(service.executable_, config.chromedriverPath);
```

Comment:
This looks like a "private" property. Not sure it's safe to be relying on it.

However, I know you're just following convention here, so no need to change it in this PR.